content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -2,7 +2,47 @@
 name: Workspace
 match: [workspace]
 tooltip: >-
-  Directory containing all your project files e.g. raw datasets, source code, ML
-  models, etc. Typically, it's also a Git repository. It will contain your DVC
+  The directory containing all your project files, e.g., the raw data, source
+  code, ML models. Typically, it's also a Git repository. It contains your DVC
   project.
 ---
+
+# Workspace
+
+A data science project can consist of data obtained from many distinct sources.
+These may be split into multiple files or directories or (as the project
+structure needs) have different versions for different requirements, e.g., a
+smaller / simplified version might be required in prototyping for faster
+feedback and shorter training times. A single workspace to manage all artifacts
```

Comment:
> or have different versions for different requirements...

Let's not go into versioning here, I think. At least not by implying they're all in the workspace because in DVC the workspace only holds one version (the rest are cached and managed via Git, metafiles, etc.

Reply:
> versioning needs and managing dependencies make it increasingly difficult

p.s. This is  better way to very subtly mention versioning (could even link to the corresponding Use Case doc).